ObjectiveSee/DoNotDisturb.munki.recipe

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Description</key>
+    <string>Downloads, unzips, verifies, packages and imports the installer of the latest version of DoNotDisturb by Objective-See into munki. The app itself is installed and uninstalled through the shell scripts.</string> 
+    <key>Identifier</key>
+    <string>com.github.its-unibas.munki.DoNotDisturb</string>
+    <key>ParentRecipe</key>
+    <string>com.github.its-unibas.pkg.DoNotDisturb</string>
+    <key>MinimumVersion</key>
+    <string>1.0.0</string>
+    <key>Input</key>
+    <dict>
+        <key>MUNKI_NAME</key>
+        <string>%NAME%</string>
+        <key>MUNKI_REPO_SUBDIR</key>
+        <string>apps/%NAME%</string>
+        <key>MUNKI_CATEGORY</key>
+        <string>Utilities</string>
+        <key>MUNKI_DEVELOPER</key>
+        <string>Objective-See</string>
+        <key>DISPLAY_NAME</key>
+        <string>%NAME%</string>
+        <key>pkginfo</key>
+        <dict>
+            <key>catalogs</key>
+            <array>
+                <string>testing</string>
+            </array>
+            <key>description</key>
+            <string>Do Not Disturb (DND) is a free, open-source utility that aims to detect and alert you of "evil maid" attacks</string>
+            <key>display_name</key>
+            <string>%DISPLAY_NAME%</string>
+            <key>name</key>
+            <string>%MUNKI_NAME%</string>
+            <key>unattended_install</key>
+            <true/>
+            <key>category</key>
+            <string>%MUNKI_CATEGORY%</string>
+            <key>developer</key>
+            <string>%MUNKI_DEVELOPER%</string>
+<!-- The following scripts install and uninstall the app by simulating a user clicking the "install" and uninstall buttons
+within the Installer app. The script has been modeled after the scripts in com.github.its-unibas.munki.OverSight -->
+            <key>postinstall_script</key>
+            <string>#!/bin/sh
+mv "/Applications/Do Not Disturb Installer.app" "/usr/local/var"
```

Comment:
This will fail if `/usr/local/var` doesn't exist. (On most machines, it doesn't exist.)

Please use [pkgcreator](https://github.com/autopkg/autopkg/wiki/Processor-PkgCreator) in combination with [pkgrootcreator](https://github.com/autopkg/autopkg/wiki/Processor-PkgRootCreator) instead of `AppPkgCreator` in your `pkg` recipe. Use it to install the app in the path of your choice. `/usr/local/var` is fine, but `/usr/local/objective-see` would work too.

This also applies for all other `pkg` recipes in this Pull Request.

Reply:
@oliverweinm 
Any news?

Reply:
Hello, your requested changes are implemented in commit b0248f :)
https://github.com/autopkg/its-unibas-recipes/pull/60/commits/b02483f6fc25cdcaed8aa36153efc2975ea25476